app/models/match.rb

```diff
@@ -2,26 +2,32 @@
 #
 # Table name: matches
 #
-#  id              :uuid             not null, primary key
-#  starts_at       :datetime
-#  casters         :jsonb
-#  data            :jsonb
-#  away_team_id    :uuid             not null
-#  home_team_id    :uuid             not null
-#  losing_team_id  :uuid
-#  winning_team_id :uuid
-#  season_id       :uuid             not null
-#  created_at      :datetime         not null
-#  updated_at      :datetime         not null
+#  id           :uuid             not null, primary key
+#  starts_at    :datetime
+#  casters      :jsonb
+#  data         :jsonb
+#  away_team_id :uuid             not null
+#  home_team_id :uuid             not null
+#  loser_id     :uuid
+#  winner_id    :uuid
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
 #
 
 class Match < ApplicationRecord
-  belongs_to :season
```

Comment:
Removed this entirely (and the foreign key) as this information can be gathered here:

```
has_one :season, through: :division
```